spec/controllers/users/delete_requests_specs.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe 'DELETE' do
+    before { sign_in user }
+    let(:user) { FactoryBot.create(:user, role: :admin) }
```

Comment:
@TehanovAnton Please configure rspec and replace `FactoryBot.create(:user, role: :admin)` with `create(:user, role: :admin)`